src/charm.py

```diff
@@ -223,6 +223,15 @@ def _on_collect_unit_status(self, event: ops.CollectStatusEvent) -> None:
 
         snap_service = get_installed_snap_service(SNAP_NAME)
 
+        # The snap_channel config option is ignored if there is a snap resource,
+        # so warn the user.
+        if self.model.config["snap_channel"] != "latest/stable" and self.get_resource():
```

Comment:
Why do we only care about the magic string `latest/stable`?

Reply:
Because `latest/stable` is the default value for snap_channel, and we don't have a generic way with the ops framework (as far as I know) to tell if the config has been user-set.

Reply:
I am assuming it's related to the unset `snap_channel` option

Reply:
> Because latest/stable is the default value for snap_channel, and we don't have a generic way with the ops framework (as far as I know) to tell if the config has been user-set.

Can we put the exact command for in the message? I guess some impatient user (like me) may accidentally use `juju config snap_channel=""`  when I see `unset`

Reply:
Good point, let me update it. :+1:

Reply:
Updated, tested, and ready for review again. :)

Reply:
thanks